clang/lib/AST/ExprConstant.cpp

```diff
@@ -11486,6 +11486,7 @@ bool IntExprEvaluator::CheckReferencedDecl(const Expr* E, const Decl* D) {
 
 /// Values returned by __builtin_classify_type, chosen to match the values
 /// produced by GCC's builtin.
+/// The values can be found in gcc/typeclass.h in the GCC repository.
```

Comment:
```suggestion

```
For licensing clarity, I don't think we want to encourage people to go look at the source of GCC even though this is a helpful comment.